src/cares_wrap.cc

```diff
@@ -150,6 +150,8 @@ class ChannelWrap : public AsyncWrap {
   void EnsureServers();
   void CleanupTimer();
 
+  void OperateActivityQueryCount(int count);
```

Comment:
nit: `Operate` seems like an odd choice of words; `modify` or maybe `add` sound more common?